esp/esp/django_settings.py

```diff
@@ -369,6 +371,7 @@ def custom_show_toolbar(request):
         ('esp.utils.shell_utils', '*'),
         )
 
+#   Set test runner to behave like pre-1.6 versions of Django
 #   Exclude apps from testing
 TEST_RUNNER = 'esp.utils.testing.ExcludeTestSuiteRunner'
```

Comment:
Note that we will need to update this for 1.8; I'm happy to not worry about it for this PR though.